api-reference/receipts/get-started.markdown

```diff
@@ -318,7 +318,7 @@ curl -v -k -X POST https://us.api.concursolutions.com/receipts/v4/users/{USER ID
 -H "Content-Type:multipart/form-data" \
 -H "link: <http://schema.concursolutions.com/{VALIDATION SCHEMA FROM SCHEMA ENDPOINT}.schema.json>;rel=describedBy" \
 -F "receipt=<{PATH TO YOUR RECEIPT JSON};type=application/json" \
--F "image=@{PATH TO YOUR IMAGE};type=image/{FILE EXTENSION OF YOUR IMAGE}"
+-F "image=@{PATH TO YOUR IMAGE};type={FILE EXTENSION OF YOUR IMAGE}"
```

Comment:
Should this be changed to... file mime type? Would that make more sense than saying file extension, which implies _only_ `gif`, `png`, etc.